Project.toml

```diff
@@ -4,6 +4,7 @@ author = ["Mauro Werder <mauro3@runbox.com>"]
 version = "0.12.0"
 
 [deps]
+ConstructionBase = "187b0558-2788-49d3-abe0-74a17ed4e7c9"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 
 [compat]
```

Comment:
How about adding `ConstructionBase = "0.1, 1.0"`? (I think CompatHelper can figure out 1.0 but I'm not sure if it adds 0.1)